static/src/javascripts/projects/common/modules/ui/subscription-banner.js

```diff
@@ -212,21 +231,26 @@ const show: () => Promise<boolean> = async () => {
     return Promise.resolve(true);
 };
 
-const canShow: () => Promise<boolean> = () => {
+const canShow: () => Promise<boolean> = async () => {
+    const hasAcknowledgedSinceLastRedeploy = await hasAcknowledgedBanner(
```

Comment:
Just for the record, not one to fix now, but I'd probably chain an extra promise rather than introduce async await here.